.circleci/config.yml

```diff
@@ -2,12 +2,13 @@ version: 2
 jobs:
   test:
     docker:
-      - image: nathanielc/flux-build:latest
+      - image: nathanielc/flux-build:rust
```

Comment:
Should you change this to `latest` given that we are merging to master?

Or should that be done after this gets merged with a separate PR 🤔

Reply:
It needs to be done after we merge to master. Its kind of a chicken and the egg. I'll create a PR to do that now so we don't forget.